handler/handler.go

```diff
@@ -450,6 +356,7 @@ func generatePrivateURL(downloadServiceURL, instanceID string) string {
 }
 
 // generateS3Filename generates the S3 key (filename including `subpaths` after the bucket) for the provided instanceID
+// TODO filename should be datasets/<dataset_name>_<version>.csv to match CMD naming
```

Comment:
Should this be created as a ticket to ensure it gets picked up?

Reply:
Maybe yes, I started working on making the changes to consume the new topic and schema generated by dataset api (when Neil finishis his changes) - maybe the file renaming should be part of the same ticket?

Reply:
created the following trello card: https://trello.com/c/lLu8DzMx/5398-cantabular-csv-exporter-should-consume-cantabular-export-start-topic